### PR TITLE
Neo4j python implementation

### PR DESCRIPTION
**Pull request by yc4007**
Neo4j python implementation for graph generation

### CODE REVIEW

File: CommunityContribution.Rmd [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (kq2153) [inline CommunityContribution.Rmd:1]: Please delete this line (the file should not have "---" in it

File: CommunityContribution.Rmd [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: "Community Contribution"
```
Comment (kq2153) [inline CommunityContribution.Rmd:2]: The first line of the document should start with a single "#" followed by a single whitespace, followed by your project title. 
e.x) # Neo4j python implementation

File: CommunityContribution.Rmd [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+author: Yuxin Cui
```
Comment (kq2153) [inline CommunityContribution.Rmd:3]: The line after the title should be blank
The 3rd line is your name (without the "author:")

File: CommunityContribution.Rmd [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+html_document: default
+---
+
+```{r setup, include=FALSE}
```
Comment (kq2153) [inline CommunityContribution.Rmd:7]: please delete the "setup": 
this should be ```{r, include = FALSE}```

File: CommunityContribution.Rmd [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Community Contribution"
+author: Yuxin Cui
+html_document: default
```
Comment (kq2153) [inline CommunityContribution.Rmd:4]: You don't need this

**Review by kq2153 — CHANGES_REQUESTED** [review]
Please make changes to address the comments made. Thanks!

**Comment by kq2153** [comment]
@yc4007 Hi, would you please be able to make the changes requested? This cannot be merged until those changes are made. Thanks!

**Comment by kq2153** [comment]
@yc4007 Hi, just checking in but would you be able to make the edits suggested above? Once these format changes are made, the file can be safely merged into the rest of the CC bookdown. Thanks!

**Comment by yc4007** [comment]
Sure I will change them right now.

**Comment by yc4007** [comment]
Hi,

Thank you for the reminding email. I have forced pull my newest changes to
branch py2neo. Could you help me delete my old branch neo4j?

Thank you!

**Comment by kq2153** [comment]
Hi, sorry just saw this. It shows that the pull request is coming from py2neo now so I think it's fine unless there is another pull request coming from your old branch as well. 

Also @yc4007 would you be able to rename your .Rmd file? The requirements for the filename are:

-  words only and joined with underscores, no white space
- all lowercase letters (e.g neo_python_implementation.Rmd)

Also for the author name, it only needs to be your name; you can remove the "author:" 

Thanks!

**Comment by kq2153** [comment]
@yc4007 Hi, would you be able to make the 2 last changes (renaming the RMD file and fixing the author line")? Thank you!

**Comment by yc4007** [comment]
Geez yes sorry for the late response. I force pushed again. Let me know if
you are able to see it.
My apologies,
Yuxin

**Pull request merged**